### ISSUE TITLE
Condition always active: Quest Event

### ISSUE THREAD

**Issue by SayuShira** [MidoriKami/ChillFrames#13]
The Quest Event Condition shows status "Active" all the time. Noticed it while crafting, but is also still active after switch zone and job.

This is new, likely due to the Hotfix just now. Wasn't sure where to put it, but as I saw it in ChillFrames, so I'm letting you know ❤️

**Comment by SayuShira** [comment]
To supplement some info: I got a Quest active, a Mandervile Upgrade. No CRP. Restarted the game. Didn't quest in this session, just loaded in. It shows Quest Event Active permanently.
Abandoned the quest, still in a Quest Event.

I started a Leve but that seems to be a Duty, not a Quest Event, so it didn't change anything. Don't exactly know how to trigger a Quest Event to see if its just weirdly stuck on my end, since Devs said on their end it looks normal.
